WalletWasabi.Gui/Shell/Commands/DiskCommands.cs

```diff
@@ -68,19 +69,19 @@ private void OnOpenWalletsFolder()
 			IoHelpers.OpenFolderInFileExplorer(Global.WalletsDir);
 		}
 
-		private void OnOpenLogFile()
+		private Task OnOpenLogFile()
 		{
-			FileHelpers.OpenFileInTextEditor(Logger.FilePath);
+			return FileHelpers.OpenFileInTextEditorAsync(Logger.FilePath);
```

Comment:
async await

Reply:
It is unnecessary because there is no continuation and implies a performance penalty (not in this specific case but in general):

* https://stackoverflow.com/questions/35201881/should-i-use-async-await-for-every-method-that-returns-a-task
* https://blog.stephencleary.com/2016/12/eliding-async-await.html
* https://docs.microsoft.com/en-us/archive/msdn-magazine/2011/october/asynchronous-programming-async-performance-understanding-the-costs-of-async-and-await#know-when-not-to-use-async

--- 

Anyway, I will add the `async/await`.

Reply:
We've decided we don't do this, but I don't remember the reasoning. However let's decide on it before merging this PR. @molnard was the one who researched the question previously and came to the conclusion we must do async await even in this situation. @molnard can you chime in?

Reply:
I have already made the changes.